ACE/ace/Auto_Ptr.h

```diff
@@ -68,9 +68,14 @@ class ACE_Auto_Basic_Ptr
 
 ACE_END_VERSIONED_NAMESPACE_DECL
 
+// NOTE: ACE7 and TAO3 require C++11 support or newer; jwillemsen commented on Jun 1, 2021
```

Comment:
This could be problematic at this point as ACE_Auto_Ptr is still used in some places in ACE/TAO

Reply:
OK